include/deal.II/base/data_out_base.h

```diff
@@ -3333,7 +3338,8 @@ class XDMFEntry
    * If the entry is not valid, this returns an empty string.
    */
   std::string
-  get_xdmf_content(const unsigned int indent_level) const;
+  get_xdmf_content(const unsigned int         indent_level,
+                   const ReferenceCell::Type &reference_cell_type) const;
```

Comment:
This looks like a public function. I guess we have to keep the original function which takes a single parameter (we can deprecate it) and add one which takes two.